src/styles/bloks/card.scss

```diff
@@ -0,0 +1,100 @@
+@use '../utils/variables' as *;
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: $card-width;
+  height: $card-height;
+  background-color: white;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-left: 19px;
+    object-fit: fill;
+  }
+
+  &__description {
+    margin-left: $card-margin;
```

Comment:
The variable '$card-margin' is used here. Make sure it is defined in the 'variables' file. If not, define it or import it from the correct file.